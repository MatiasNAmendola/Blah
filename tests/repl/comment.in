# this should be ignored
if True then
    'You should see this'
    # but not this
    #because this is a comment
    'You should also see this'#but not his
    '
Hash signs in strings #have
#no effect'#but this does
otherwise
    'You should not see this'
fi

#
a: [1, 2, #3,
    4, 5]
a

