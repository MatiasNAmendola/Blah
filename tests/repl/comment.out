You should see this
You should also see this

Hash signs in strings #have
#no effect
[1, 2, 4, 5]
